data/changelogs/ddos-http.yaml

```
@@ -4,11 +4,14 @@ productName: DDoS protection
 subCategoryName: HTTP
 topLevelLink: "/ddos-protection/change-log/"
 entries:
-- publish_date: '2024-03-12'
-  scheduled_date: "2024-04-04"
+- publish_date: '2024-04-19'
+  scheduled_date: "2024-04-29"
```

```suggestion
  scheduled_date: '2024-04-29'
```
For the sake of consistency.